.github/workflows/pahole.yml

```diff
@@ -0,0 +1,40 @@
+name: pahole-staging
+
+on:
+  schedule:
+    - cron:  '0 * * * *'
+
+
+jobs:
+  vmtest:
+    runs-on: ubuntu-latest
+    name: Kernel LATEST + staging pahole
+    env:
+      STAGING: tmp.master
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/setup
+      - name: Get current pahole sha
+        id: current_sha
+        run:
+          git ls-remote https://git.kernel.org/pub/scm/devel/pahole/pahole.git $STAGING | awk '{print "::set-output name=sha::" $1}'
+      - name: Get latest result for this sha
+        id: latest
+        uses: pat-s/always-upload-cache@v2
+        with:
+          path: last_tested_pahole
+          key: ${{ steps.current_sha.outputs.sha }}
+      - name: Return cached test result
+        run: exit `cat last_tested_pahole`
+        if: steps.latest.outputs.cache-hit == 'true'
+      - uses: ./.github/actions/vmtest
+        with:
+          kernel: LATEST
+          pahole: $STAGING
+        if: steps.latest.outputs.cache-hit != 'true'
+      - name: Save success
+        run: echo 0 > last_tested_pahole
+        if: steps.latest.outputs.cache-hit != 'true'
```

Comment:
do I understand correctly that we'll poll latest tmp.master SHA every hour, and will run full test only if it changes from the last time, right? That's pretty nice!

Reply:
Yes, that is how it supposed to work.